templates/15-rust.sh

```diff
@@ -84,8 +84,6 @@ build_rust_build() {
 	local manifest
 	manifest="$(cargo read-manifest --manifest-path "$SRCDIR"/Cargo.toml || true)"
 	abjson_get_item "${manifest}" "['targets'][0]['kind'][0]" _JSON_OUTPUT || true
-	local workspace_flag
-	workspace_flag='--workspace'
```

Comment:
I don't get why this was moved down.

Reply:
Because why not? Variable definitions should be close to where they are used, or be at the top of the function block.

Reply:
> Because why not? Variable definitions should be close to where they are used, or be at the top of the function block.

It seems that you are defending unexplained changes,